package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gofynd/fdk-cli",
-  "version": "4.0.0-beta.1",
```

Comment:
@Shivraj0 change this 4.0.0-beta.1 only, as there is no change related to fdk build.

Reply:
Why was this changes to `4.0.0-beta.1` at first place?